core/module/modular.go

```diff
@@ -52,6 +54,12 @@ type Authorizer interface {
 	Modular
 	// VerifyAuthorize verifies the operator authority.
 	VerifyAuthorize(ctx context.Context, auth AuthOpType, account, bucket, object string) (bool, error)
+	// GetAuthNonce get the auth nonce for which the Dapp or client can generate EDDSA key pairs.
+	GetAuthNonce(context.Context, *gfspserver.GetAuthNonceRequest) (*spdb.OffChainAuthKey, error)
```

Comment:
core dir should use the interface, include the func params
core dir should not import base/types/gfspserver pkg.
suggest to refine gfspserver.GetAuthNonceRequest to interface and gfspserver.GetAuthNonceRequest implement the interface

Reply:
Done